terraform/common/live.hcl

```diff
@@ -0,0 +1,8 @@
+locals {
+  account_id  = "638924580364"  # auth account
```

Comment:
Could make this a named variable rather than a comment?

Reply:
`common/live.hcl` and `common/labs.hcl` contain variables that are referenced in identical way between the envs. e.g. 
* https://github.com/hmrc/aws-grant-user-access/blob/INFRA-7955-bootstrap-live/terraform/bootstrap/labs/terragrunt.hcl#L5-L12
* https://github.com/hmrc/aws-grant-user-access/blob/INFRA-7955-bootstrap-live/terraform/bootstrap/live/terragrunt.hcl#L5-L12

A named variable such as `auth_account_id` will only be relevant in the context of the live environment, and may present a small problem to labs env since it is not needed in labs.